librarymanagement/src/main/scala/sbt/librarymanagement/Resolver.scala

```diff
@@ -174,7 +174,9 @@ object Resolver {
   private[sbt] def useSecureResolvers = sys.props.get("sbt.repository.secure") map { _.toLowerCase == "true" } getOrElse true
 
   val TypesafeRepositoryRoot = typesafeRepositoryRoot(useSecureResolvers)
-  val SbtPluginRepositoryRoot = sbtPluginRepositoryRoot(useSecureResolvers)
+  val SbtRepositoryRoot = sbtPluginRepositoryRoot(useSecureResolvers)
```

Comment:
Should we rename `def sbtPluginRepositoryRoot(...)` to `def sbtRepositoryRoot(...)`